grails-app/views/inventoryItem/_showStockHistory.gsp

```diff
@@ -340,6 +340,26 @@
 
                                 <g:if test="${stockHistoryEntry?.showDetails}">
                                     <div >
+                                        <g:if test="${stockHistoryEntry?.transaction?.order?.code == org.pih.warehouse.order.OrderTypeCode.PURCHASE_ORDER.name()}">
```

Comment:
You are importing `OrderTypeCode` at the top. `OrderTypeCode.PURCHASE_ORDER.name()` should work.

Reply:
done